dockerfiles/image.sh

```diff
@@ -1,11 +1,22 @@
 #!/bin/bash
-set -e
+
+set -eEuo pipefail
+
+trapFunc() {
+  local lineno="$1"
+  local code="$2"
+  local command="$3"
+  echo "error exit status $code, at file $0 on or near line $lineno: $command"
+}
+trap 'trapFunc "${LINENO}/${BASH_LINENO}" "$?" "$BASH_COMMAND"' ERR
 
 globals() {
     SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}"; )" &> /dev/null && pwd 2> /dev/null; )"
     ROOT_DIR=$(git rev-parse --show-toplevel)
 
+    set +e
     export $(grep -v '^#' $ROOT_DIR/.env | sed -e 's/\r$//' | xargs) > /dev/null
```

Comment:
@GordonSmith - wasn't entirely sure if this was right, but it throws errors (when either .env is missing, or if it just contains comments, didn't try other combinations). But because of the possible errors, I enclosed it with error suppression.

Reply:
I think that is ok.